src/chrome/content/rules/Yeti-designs.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Yeti-designs.com">
+	<target host="yeti-designs.com" />
```

Comment:
I get connection refused here. Could you add a redirect rule to `www` and a comment mentioning this please?